libr/bin/p/bin_elf.c

```diff
@@ -47,6 +47,7 @@ static void * load_bytes(RBinFile *arch, const ut8 *buf, ut64 sz, ut64 loadaddr,
 		return NULL;
 	}
 	tbuf = r_buf_new ();
+	tbuf->fd = arch->fd;
```

Comment:
better not to force this here. we should work with rbuffers independely from the data source baked behind the implementation